src/PythonMigrationViewExtension/Controls/MigrationAssistantDisclaimer.xaml

```diff
@@ -0,0 +1,109 @@
+﻿<Window x:Class="Dynamo.PythonMigration.Controls.MigrationAssistantDisclaimer"
+        x:ClassModifier="internal"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:ui="clr-namespace:Dynamo.UI;assembly=DynamoCoreWpf"
+        xmlns:p="clr-namespace:Dynamo.PythonMigration.Properties;assembly=PythonMigrationViewExtension"
+        xmlns:local="clr-namespace:Dynamo.PythonMigration.Controls"
+        Width="660"
+        SizeToContent="Height"
+        ResizeMode="NoResize"
+        mc:Ignorable="d"
+        WindowStartupLocation="CenterScreen"
+        Title="Migration Assistant Warning">
```

Comment:
The window title should probably say `Disclaimer` but, most importantly,  it should be a resource.

Now that I notice, some additions from previous PRs are also missing this:
- `Migration Assistant` in `BaseDiffViewer`
- `Before -> After` in `InLineControl`
- `Before` and `After` in `SideBySideControl`

Can I ask you to externalize those as resources too?